include/swift/AST/ParseRequests.h

```diff
@@ -117,6 +121,9 @@ class CodeCompletionSecondPassRequest
   // Evaluation.
   bool evaluate(Evaluator &evaluator, SourceFile *SF,
                 CodeCompletionCallbacksFactory *Factory) const;
+
+public:
+  evaluator::DependencySource readDependencySource(Evaluator &) const;
```

Comment:
We should never emit dependencies while doing code completion, so this doesn't seem necessary.

Reply:
Code completion runs with `-primary-file` turned on, which creates referenced name trackers even if the results aren't being written anywhere.